packages/api/src/controllers/project.ts

```diff
@@ -177,4 +181,115 @@ app.delete("/:id", authorizer({}), async (req, res) => {
   res.end();
 });
 
+app.post(
+  "/job/projects-cleanup",
+  authorizer({ anyAdmin: true }),
```

Comment:
Btw if you're creating this only because the other jobs have sth like it, it's unnecessary! The only reason they're like that is tech debt: we already had such APIs that were called from GitHub actions.

The cleanest way for a new job IMO will be to implement everything on the job. Then if later we want the API to trigger it manually, we can make an API that calls the job function (not the other way around).

Your call though, especially if you're already too far into having this API that it would be a big refactor to get rid of it now.